org.scala-ide.sdt.core/src/org/scalaide/ui/internal/editor/ScalaContentProvider.scala

```diff
@@ -0,0 +1,86 @@
+package org.scalaide.ui.internal.editor
+import org.eclipse.jface.viewers.ITreeContentProvider
+import org.eclipse.jface.viewers.Viewer
+
+
+
+
+/**
+ * @author ailinykh
+ */
+class ScalaContentProvider extends ITreeContentProvider {
```

Comment:
`ScalaContentProvider` is not a good name for this class, since it is only related to the Outline View.